installer/ansible/script/set_nginx_config.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
```

Comment:
this file may not require. Please check

Reply:
used by dashboard-installer
https://github.com/sodafoundation/multi-cloud/blob/b6459548812ed643e9a0e8f1186f5d341e86cab2/installer/ansible/roles/dashboard-installer/scenarios/source-code.yml#L41-L43